server/utils/placementRequests/table.ts

```diff
@@ -0,0 +1,59 @@
+import { add } from 'date-fns'
+import { PlacementCriteria, PlacementRequest } from '../../@types/shared'
+import { TableCell, TableRow } from '../../@types/ui'
+import { DateFormats } from '../dateUtils'
+import { sentenceCase } from '../utils'
+
+export const DIFFERENCE_IN_DAYS_BETWEEN_DUE_DATE_AND_ARRIVAL_DATE = 7
+
+export const tableRows = (placementRequests: Array<PlacementRequest>): Array<TableRow> => {
+  return placementRequests.map((placementRequest: PlacementRequest) => {
+    return [
+      nameCell(placementRequest),
+      crnCell(placementRequest),
+      dueDateCell(placementRequest, DIFFERENCE_IN_DAYS_BETWEEN_DUE_DATE_AND_ARRIVAL_DATE),
+      expectedArrivalDateCell(placementRequest),
+      locationCell(placementRequest),
+      essentialPlacementCriteriaCell(placementRequest),
+      desirablePlacementCriteriaCell(placementRequest),
+    ]
+  })
+}
+
+const nameCell = (placementRequest: PlacementRequest): TableCell => ({ text: placementRequest.person.name })
```

Comment:
Wonder whether we can share some of these things across various different utils?